backend/src/main/java/hu/flowacademy/worksheet/exceptionHandler/CustomExceptionHandler.java

```diff
@@ -0,0 +1,29 @@
+package hu.flowacademy.worksheet.exceptionHandler;
+
+
+import hu.flowacademy.worksheet.exception.WorksheetUserException;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.context.request.WebRequest;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@ControllerAdvice
+public class CustomExceptionHandler {
+    @ExceptionHandler(Exception.class)
+    public final ResponseEntity<Object> handleAllExceptions(Exception e, WebRequest request) {
+        List<String> messageList = new ArrayList<>();
+        messageList.add(e.getLocalizedMessage());
+        return new ResponseEntity<>(messageList, HttpStatus.INTERNAL_SERVER_ERROR);
```

Comment:
Ha String listaként akarod visszaadni, akkor nyugodtan használj List.of(...)-ot és akkor külön változó deklarációra sincs szükség.